graphene_django/compat.py

```diff
@@ -1,5 +1,6 @@
 class MissingType(object):
-    pass
+    def __init__(self, *args, **kwargs):
```

Comment:
Not sure why this is needed?

Reply:
When you are initializing things like the ArrayField it will send some args to the constructor that by default the MissingType is not expecting and therefore will fail saying that the Field was overloaded